packages/node/src/indexer/dictionary/v2/ethDictionaryV2.ts

```diff
@@ -146,15 +148,15 @@ function eventFilterToDictionaryCondition(
 function sanitiseDictionaryConditions(
   dictionaryConditions: EthDictionaryV2QueryEntry,
 ): EthDictionaryV2QueryEntry {
-  if (!dictionaryConditions.logs.length) {
+  if (dictionaryConditions.logs?.length === 0) {
```

Comment:
I think this is a behaviour change. If `logs` is undefined then it wont be removed.

```suggestion
  if (!dictionaryConditions.logs?.length) {
```